clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -2262,6 +2286,27 @@ public void onComplete(Map<TopicPartition, OffsetAndMetadata> offsets, Exception
         };
     }
 
+    private OffsetCommitCallback callback(final Map<TopicPartition, OffsetAndMetadata> expectedOffsets,
+                                          final AtomicBoolean success) {
+        return new OffsetCommitCallback() {
+            @Override
+            public void onComplete(Map<TopicPartition, OffsetAndMetadata> offsets, Exception exception) {
+                if (expectedOffsets.equals(offsets) && exception == null)
+                    success.set(true);
+            }
+        };
+    }
+
+    private OffsetCommitCallback callback(final String expectedMessage, final AtomicBoolean success) {
```

Comment:
This new function seems not used anywhere?

Reply:
Yea, I forgot to delete that